knora/dsplib/schemas/data.xsd

```diff
@@ -409,7 +409,9 @@
         <xs:attribute name="label" type="xs:string" use="required"/>
         <xs:attribute name="restype" type="xs:string" use="required"/>
         <xs:attribute name="id" type="xs:ID" use="required"/>
+        <xs:attribute name="iri" type="xs:string" use="optional"/>
         <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
+        <xs:attribute name="ark" type="xs:string" use="optional"/>
```

Comment:
I haven't worked with XML Schema in a while, so I'm not even sure that would be possible... but it would be neat to check, if `ark` is present, and if so, not allow `iri`.

Reply:
That would be possible with XMLSchema 1.1 but unfortunately the python module we are using for schema validation doesn't support 1.1 as it seems. So, I leave it to that.